package.json

```diff
@@ -16,5 +16,438 @@
     "eslint-plugin-prettier": "^3.3.1",
     "jest": "^26.6.3",
     "prettier": "^2.2.1"
-  }
+  },
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
+    "abab": "^2.0.6",
```

Comment:
I have no idea why all this is here! You dont need to add any additional packages as everything you need is already in the repo, these should all be removed